internal/networkmap/update_identity.go

```diff
@@ -44,6 +44,11 @@ func (nm *networkMap) updateIdentityID(ctx context.Context, id *fftypes.UUID, dt
 		return nil, i18n.NewError(ctx, coremsgs.Msg404NoResult)
 	}
 
+	// TODO is this right ? code below assumes this is true and errors otherwise
```

Comment:
This is only true for Node based identity I believe - but would need to double check